server/queries/createSurvey.js

```diff
@@ -0,0 +1,24 @@
+const createSurvey = async (
+  surveyObj,
+  newQuestionsArray,
+  surveysCollection,
```

Comment:
same for this one. Call getDb() and get collection 'surveys', instead of doing in the handler 👍 remove second argument surveysCollection